components/GroupProfilePanels/GroupSettingsPanel.js

```diff
@@ -0,0 +1,245 @@
+      submitting: false,
+      validity: {
+        address: true,
+        description: true,
+        discoverable: true,
+        name: true,
+        slug: true,
+      },
+    }
+  }
+
+  render () {
+    const { group } = this.props
+    const {
+      changes,
+      submitting,
+    } = this.state
+
+    const address = changes.address || group.attributes.address
+    const description = changes.description || group.attributes.description
+    // const discoverable = changes.discoverable || group.attributes.discoverable
+    const name = changes.name || group.attributes.name
+    // const slug = changes.slug || group.attributes.slug
+
+    return (
+      <section className="settings">
+        <form onSubmit={this._handleSubmit}>
+          <fieldset>
+            <label htmlFor="name">
+              Group name
+            </label>
+
+            <ValidatedInput
+              data-pattern-explainer="Please make sure to only use letters, numbers, hyphens (-), and underscores (_)"
+              disabled={submitting}
+              id="name"
+              name="name"
+              onChange={this._handleChange}
+              pattern="[\w\s_-]+"
+              placeholder="e.g. Quigley's Tavern"
+              required
+              type="text"
+              value={name} />
+          </fieldset>
+
+          {/* <fieldset>
+            <label htmlFor="slug">Permalink</label>
+
+            <div className="input-group">
+              <label htmlFor="slug">
+                https://rollforguild.com/groups/
+              </label>
+
+              <input
+                disabled={submitting}
+                id="slug"
+                name="slug"
+                onChange={this._handleSlugChange}
+                pattern="[\w\s_-]+"
+                placeholder={convertStringToSlug(name)}
+                type="text"
+                value={slug} />
+            </div>
+          </fieldset> */}
+
+          <fieldset>
+            <label htmlFor="description">
+              Group description
+            </label>
+
+            <textarea
+              aria-describedby="description"
+              disabled={submitting}
+              id="description"
+              maxLength={1000}
+              name="description"
+              onChange={this._handleChange}
+              placeholder="Tell your members what you'll be playing, or maybe a bit about your GM style."
+              value={description} />
+
+            <small>Tell your members what you'll be playing, or maybe a bit about your GM style.</small>
+          </fieldset>
+
+          <fieldset>
+            <label htmlFor="address">
+              Where will you be playing?
+            </label>
+
+            <AddressInput
+              disabled={submitting}
+              id="address"
+              onChange={this._handleAddressChange}
+              required
+              value={address} />
+          </fieldset>
+
+          {/* <fieldset className="horizontal">
```

Comment:
I take it we're still waiting on the API fix for this?

Reply:
Yup, discoverability still doesn't come back from the API.